### DISCUSSION TITLE
Feature Request: Expand Collapse Block, Tab Block, Image Options

### DISCUSSION THREAD

**Discussion by jonnymccullagh** [outlines-dev/outlines#1112] [Feature requests]
I have begun using Outline and there are one or two things I think would be good to have:

1. Expand/Collapse block - if I have a large block of code I would prefer to put it inside a container that is collapsed by default and if the reader wants to examine the code they can expand the block. 
2. Tab Block - would it be possible to add a block that renders with tabs. My use case is that I would like to show commands in different languages. So I might have a python/javascript alternative, or a bash/fish alternative, or docker/compose/k8s alternative.
3. Image Border - I also like to add a border around all the screenshots I add to make them more distinguishable as screenshots. I'd love an icon beside the alignment buttons for adding a border.
4. Image size - I also like to reduce the size of my screenshots (again to make them more distinguishable that they are not something from the OS itself). A button beside the 'full width' button that reduces the size by 100px each click would be nice. I know I can use the handles around the image but these are difficult to grab hold of and a button would be easier/nicer.

**Comment by rlouf** [comment]
Hi!

I think you wrote on the wrong Outline repo :)

**Comment by jonnymccullagh** [comment]
oops
